manage/sawtooth_manage/subproc.py

```diff
@@ -60,6 +61,10 @@ def start(self, node_args):
         commands = ['validator'] + state['Processors']
         if node_args.genesis:
             commands = ['sawtooth'] + commands
+            # clean data dir of existing genesis node artifacts
+            data_dir = os.path.join(os.path.expanduser("~"), 'sawtooth', 'data')
```

Comment:
@peterschwarz It comes from the same location that the SubprocessNodeController gets the location of the state_dir, and seems to be the design. See line: 31